### ISSUE TITLE
Normalização dos dados

### ISSUE THREAD

**Issue by karlvmsousa** [mmp-s/mining-project#4]
Para um melhor tratamento dos dados, é necessário fazer uma normalização durante o pré-processamento. Nesse caso deve-se avaliar qual a melhor técnica a ser aplicada. Alternativas:
- sklearn.preprocessing.MinMaxScaler
- sklearn.preprocessing.StandardScaler (por default define média 0 e desvio padrão 1)
